setup.py

```diff
@@ -55,7 +55,7 @@
     'parsing~=2.0',
     'psutil~=5.8',
     'setproctitle~=1.2',
-    'setuptools_scm~=6.0',
+    'setuptools_scm~=6.0,!=6.3.0',
```

Comment:
Curious, what's wrong with 6.3.0?

Reply:
I think it's [missing a dependency](https://github.com/pypa/setuptools_scm/issues/625)